### PR TITLE
[WIP] Add Segmentation Vignettes

### PR DESCRIPTION
**Pull request by ambrosejcarr**
## Objectives: 
- Understand the current limitations of starfish's segmentation module
- Frame feature additions which would improve functionality and ease of use for segmentation

I've put this up for @dganguli 's visibility since this might be useful for the paper. 

## Current Status
Create `SegmentationMaskCollection` objects from the following examples:
- Example of training and then applying a pre-trained Ilastik module and reading the resulting data
- Example of creating and loading ROIsets from FIJI, based on starfish data
- Example of creating and loading Masks from FIJI, based on starfish data
- Example of loading geoJSON from @berl

## Next
- Update #1321 based on learnings here. 
- Seek feedback on scoped work in #1321

### CODE REVIEW

**Comment by dganguli** [comment]
Can you put the files:
SongLinROIS_deduplicated.json
mFISH__001_registered-downsampled.png 

on S3? Worried that slack isn't forever.

**Comment by dganguli** [comment]
For manual segmentation, using FIJI Segmentation manager, the segmentation results look "off". As though it's missing data at the boundaries -- can you verify that this is a segmentation error and not an error in SegmentationMaskCollection?

**Comment by dganguli** [comment]
ROI manager may have same aforementioned boundary problem?

**Comment by dganguli** [comment]
should verify starfish.display works on images an order of magnitude bigger than we're used to -- relevant to geo_json example in this notebook

**Comment by dganguli** [comment]
@ambrosejcarr manual notebook demonstrates 3 different ways of turning manual segmentations into SegmentationMaskCollections. Ilastik notebook shows 1 way. All of these look good/useful -- can we write an API doc for exactly what we want based on these 4 methods and dump it here for review.
